docs/configuration.md

```diff
@@ -87,25 +87,45 @@ Supported placeholders/variables are:
  1. **{locale\_with\_underscore}** same as above except all hyphens '-' will be replaced with underscores '_'. This is typically used in properties and gettext projects.
  1. **{extension}** the source document file extension
 
-For example, given a source document is found at `/home/user/myproject/src/main/resource/message.properties` 
+For example, if your file structure is like following (where `{projectRoot}` is the root of your project and where zanata.xml lives):
 
-... and your source document root (src-dir option) is set to "." 
-
-... Your zanata.xml is located at `/home/user/myproject` 
+```
+{projectRoot}/
+              templates/messages/kdeedu/kalzium.pot
+              templates/messages/kdeedu/artikulate.pot
+              de-DE/messages/kdeedu/kalzium.po
+              de-DE/messages/keeedu/artikulate.po
+              ...
+              zanata.xml
+```
 
-... For a locale mapping defined as `<locale map-from="zh-CN">zh</locale>`: 
+Here we have two "pot" files as source documents and two "po" files as "de-DE" language translation documents.
 
+You can then use below configuration:
+{% highlight xml %}
+<src-dir>templates</src-dir>
+<trans-dir>.</trans-dir>
+<rules>
+    <rule pattern="**/*.pot">{locale}/{path}/{filename}.po</rule>
+</rules>
+{% endhighlight %}
 
-The following placeholders will be detected:
+Explaination: Since you define your `<src-dir>` as `templates`, file `templates/messages/kdeedu/kalzium.pot` will be mapped to a relative path file name which is `messages/kdeedu/kalzium.pot`. It then will be partitioned into several tokens to form the variables:
```

Comment:
Explanation: Since you have defined `<src-dir>` as "templates", the source document `templates/messages/kdeedu/kalzium.pot` will have its path extracted relative to `{projectRoot}/templates`, which gives the relative path `messages/kdeedu/kalzium.pot`. The relative path then will be partitioned into several tokens to form the following variables: